### PR TITLE
Train-3 Fixed date-filiter and login is responsive, forget is fixed.

### PR DESCRIPTION
**Pull request by BurleVineeth**

### CODE REVIEW

File: src/app/authentication/login/login.component.scss [hunk 1]
```diff
@@ -1,49 +1,157 @@
...
   align-items: center;
-  flex-direction: column;
+  .block {
+    padding: 30px 10px;
+    border-radius: 10px;
 
-  .container {
-    .row {
+    .title {
+      font-family: "Poppins";
```
Comment (Reetika-147daffodil) [inline src/app/authentication/login/login.component.scss]: We have globally assigned font-family , no need to use it here.

File: src/app/authentication/login/login.component.scss [hunk 2]
```diff
@@ -1,49 +1,157 @@
...
       }
+    }
+
+    form {
+      padding: 10px;
+
+      .field {
+        margin: 10px;
+        padding: 0 30px;
+        // line-height: 0;
```
Comment (Reetika-147daffodil) [inline src/app/authentication/login/login.component.scss]: remove commented code

File: src/app/authentication/login/login.component.scss [hunk 3]
```diff
@@ -1,49 +1,157 @@
...
+        span {
+          color: red;
+          margin: 5px 0 0 30px;
+        }
+
+        .red {
+          margin-top: 0px;
+        }
+
+        .full-width {
```
Comment (Reetika-147daffodil) [inline src/app/authentication/login/login.component.scss]: use global class for 100% width

File: src/app/authentication/login/login.component.scss (hunk 2 above)
Comment (BurleVineeth) [reply src/app/authentication/login/login.component.scss]: removed the commented code

File: src/app/authentication/login/login.component.scss (hunk 3 above)
Comment (BurleVineeth) [reply src/app/authentication/login/login.component.scss]: removed the full width class

File: src/app/authentication/login/login.component.scss (hunk 1 above)
Comment (BurleVineeth) [reply src/app/authentication/login/login.component.scss]: removed the font-family

**Review by Reetika-147daffodil — APPROVED** [review]

**Review by prafullcodes — APPROVED** [review]

**Pull request merged**